test/test_sample.py

```diff
@@ -604,9 +613,9 @@ def test_validate_big_run(self):
         self.assertEqual(fit.step_size.shape, (2,))
         self.assertEqual(fit.metric.shape, (2, 2095))
         self.assertEqual((1000, 2, 2102), fit.draws().shape)
-        phis = fit.draws_pd(params=['phi'])
+        phis = fit.draws_pd(vars=['phi'])
```

Comment:
should we also now test with `vars='phi'` since that is allowed? More of a general comment for all tests of these kinds

Reply:
string keyword is tested earlier, line 564